qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -67,7 +70,13 @@ def qapplication():
     app = QApplication.instance()
     if app is None:
         QCoreApplication.setAttribute(Qt.AA_ShareOpenGLContexts)
-        app = QApplication(['Mantid Workbench'])
+        argv = sys.argv[:]
+        argv[0] = 'Mantid Workbench' # replace application name
```

Comment:
Should we set `APPNAME=mantidworkbench` in `config`and use this here?

Reply:
I'm not sure how `sys.argv[0]` is used anywhere and if it is an issue that we replace it. It is normally the name of the executable. I only see the name in the frame of the splashscreen and in the system pager.

Reply:
I was just hoping to reduce the number of places we specify things like this...